config/config.go

```diff
@@ -128,6 +128,9 @@ func (cfg *AnalyzerConfig) Validate() error {
 
 // ServerConfig contains the API server configuration.
 type ServerConfig struct {
+	// ChainID is the chain ID (normally latest) for the server API
```

Comment:
How can the user derive/determine the ID? Let's document here.

I don't see `oasis-3` in the output of `oasis-node control status`, and the old value in indexer was hardcoded with no explanation. I also don't see `oasis-3` in any of our other repos. Is it just for internal bookkeeping? If so, can we make it simply `"damask"` to be consistent with analyzer names and external naming?

Reply:
You can get it programmatically from the [genesis document](https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go@v0.2201.11/genesis/api#Document), and it will change after each breaking upgrade.

The user shouldn't have to ever pass it in as a parameter, I think.

Reply:
Gotcha. So we probably shouldn't introduce the `chainId` setting in indexer's config yml, because we do set a node endpoint, and the endpoint [can provide the genesis file](https://github.com/oasisprotocol/oasis-indexer/blob/7a473104983c33020796c8232fb777634b921b08/cmd/generator/generator.go#L187-L187) that includes the chain ID.

Reply:
Oh I take it back. This is for the API server, who doesn't care about the node. So it makes sense to manually configure which chain it should read from the DB.